web/src/router.ts

```diff
@@ -28,6 +28,26 @@ const routes: RouteRecordRaw[] = [
     component: (): Component => import('~/views/ReposOwner.vue'),
```

Comment:
Could you rename `ReposOwner` and `repos-owner` to something like `Orgs` and `org`?

Reply:
Ahh I see good point.